### DISCUSSION TITLE
Using "test.cpp" - how to downliad "utility.h: header from giuthub ?

### DISCUSSION THREAD

**Discussion by anneranch** [lxqt/lxqt#2308] [Q&A]
I see recent changes in qterminal arguments....
I am taking my own direction to solve the multi argument issue. 
I have downloaded "test.cpp" and found  it uses "utility.h" header . 
**I am assuming it is in  "src"  folder but I am unable to open it.** 

PS it would be nice if I could just clone "test" directly from github....

**Comment by yan12125** [comment]
There is no test.cpp in qterminal. Could you give more details?
